### PR TITLE
Add Extensions for OpenJDK version to java -Xinternalversion

### LINKED ISSUE #1132: java -Xinternalversion output needs Extensions for OpenJDK version
java -Xinternalversion needs to output the Extensions for OpenJDK version, eg:
java -Xinternalversion:
Eclipse OpenJ9 OpenJDK 64-bit Server VM (1.8.0_152-b16) from linux-amd64 JRE (1.8.0_152-b16) with Extensions for OpenJDK for Eclipse OpenJ9 8.0.0.1, built on Feb 8 2018 10:12:41 by admin with g++-4.8 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Depends on https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/35

Closes #1132 

Signed-off-by: Andrew Leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

**Comment by andrew-m-leonard** [comment]
@pshipton Peter, please can you review this change?

**Comment by pshipton** [comment]
Please see the review comments in https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/35. I don't think OpenJ9 should be hard coding "Extensions for OpenJDK for Eclipse OpenJ9".

Also, I think we could improve the output by not repeating J9VERSION_STRING twice in the version output.

File: runtime/util/openj9_version.c [hunk 1]
```diff
@@ -32,9 +32,15 @@ j9print_internal_version(J9PortLibrary *portLib) {
 	PORT_ACCESS_FROM_PORT(portLib);
 
 #if defined(OPENJ9_BUILD)
-	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE (%s), built on %s %s by %s with %s\n",
+#if defined(J9JDK_EXT_VERSION)
+	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE (%s) with Extensions for OpenJDK for Eclipse OpenJ9 %s, built on %s %s by %s with %s\n",
```
Comment (pshipton) [inline runtime/util/openj9_version.c]: "Extensions for OpenJDK for Eclipse OpenJ9" is still hard coded. Please define this in `closed/openj9_version_info.h.in` and use the define here.

Comment (pshipton) [reply runtime/util/openj9_version.c]: If you removed the second insertion of J9VERSION_STRING that would also be appreciated, although unrelated to this change. There is no reason to include the exact same version twice.

**Comment by pshipton** [comment]
test compile plinux depends ibmruntimes/openj9-openjdk-jdk8#35

**Comment by pshipton** [comment]
jenkins compile plinux depends ibmruntimes/openj9-openjdk-jdk8#35

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK8-linux_ppc-64_cmprssptrs_le-OpenJ9/10/
https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK9-linux_ppc-64_cmprssptrs_le-OpenJ9/9/

**Comment by pshipton** [comment]
There is a typo in this change. I've opened #1147 to fix it.

**Pull request closed without merging**